src/stratis_cli/_actions/_physical.py

```diff
@@ -27,8 +27,21 @@
 from ._data import ObjectManager
 from ._data import Pool
 from ._data import pools
+from ._data import MODev
+from ._data import devs
+from ._formatting import print_table
 
 
+def state_num_to_string(val):
+    states = {
+        0: "Missing",
```

Comment:
Better to use a list, since these are just contiguous numeric indices.

Reply:
ok